### ISSUE TITLE
Remove SystemTextJsonPatch.AspNet 

### ISSUE THREAD

**Issue by davidfowl** [Havunen/SystemTextJsonPatch#1]
Doesn't seem like we should have this entire assembly just to make adding a JSON converter to MVC easier. There's also minimal APIs which has a different way to configure the JSON options.

**Comment by Havunen** [comment]
Yeah, I was just trying to avoid adding reference to 
```
  <ItemGroup>
    <FrameworkReference Include="Microsoft.AspNetCore.App" />
  </ItemGroup>
```
in the main project, but now after testing that System.Text.Json.Nodes does not even exists in netcore3.1 or net5.0 it should be reasonable to combine these. 

Maybe conditional compilation could be used anyway to include or exclude it. I will merge these projects

**Comment by davidfowl** [comment]
I don't think you need the reference at all, the extension method doesn't do enough to warrant another package IMO.

**Comment by Havunen** [comment]
Sure, I'm not aware of the consequences of that attribute tbh. Probably just better to document the behavior and drop the reference :)

**Comment by Havunen** [comment]
fixed in 0.0.2
